### ISSUE TITLE
Change to location of downloaded assets

### ISSUE THREAD

**Issue by brajbhushan-tripathi** [b-yp/logseq-link-to-local#22]
Is there a way I can change the location of the files being downloaded? Currently the files are stored in "assets\storages\logseq-link-to-local". I would like to keep them directly in the assets folder.

**Comment by b-yp** [comment]
Initially, I did consider this issue, but at that time, I couldn't find a suitable solution. Since it has been brought up again, I'll give it another try to see if there's an alternative solution.

**Comment by 64bakerst** [comment]
Yes, please - it would be also great if assets could be per folder as named by page

**Comment by b-yp** [comment]
> Yes, please - it would be also great if assets could be per folder as named by page

@64bakerst   It appears that your requirements and those of brajbhushan-tripathi are still not aligned.
